src/fides/api/api/v1/endpoints/health.py

```diff
@@ -128,15 +129,15 @@ async def database_health(db: Session = Depends(get_db)) -> Dict:
 async def workers_health() -> Dict:
     """Confirm that the API is running and healthy."""
     response = WorkerHealthCheck(
-        workers_enabled=False,
-        workers=[],
+        workers_enabled=False, workers=[], queue_counts={}
     ).dict()
 
     fides_is_using_workers = not celery_app.conf["task_always_eager"]
     if fides_is_using_workers:
         response["workers_enabled"] = True
         # Figure out a way to make this faster
         response["workers"] = get_worker_ids()
+        response["queue_counts"] = get_queue_counts()
```

Comment:
ah when i saw your testing steps on the plus PR i was wondering if this was new, it's great! 🎊 

i love exposing all these diagnostics